spec/support_specs/rails_manager_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'tmpdir'
+
+describe RailsManager do
+  let(:manager) { described_class.new(config, app_path: app_path) }
+
+  let(:config) { double }
+  let(:app_path) { File.join(tmp_dir, 'test/dummy') }
+
+  after do
+    FileUtils.remove_entry(tmp_dir)
+  end
+
+  describe '#on_rails', skip: 'Conflicts with manager in rspec_settings' do
```

Comment:
It seems that `RailsManager#unload_rails!` isn't fully unloads rails application. Running this test separately works fine, but running all tests at once raises issue

If we run `bundle exec rspec` 2 rails applications will be initialized (first one when this test is run, second one is when test with `:on_rails` hook is executed), and error will be raised.

I couldn't figure out how to fix this issue, maybe i/o creating hook we should just run tests on rails and non-rails application environment 🤔